src/modules/projects/components/tabs/TasksTab.tsx

```
@@ -6,9 +6,10 @@ import { X } from "lucide-react";
 
 interface TasksTabProps {
   workspaceData: WorkspaceData;
+  onTabChanges?: () => void;
 }
 
-const TasksTab: React.FC<TasksTabProps> = ({ workspaceData: workspaceData }) => {
+const TasksTab: React.FC<TasksTabProps> = ({ workspaceData: workspaceData, onTabChanges }) => {
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)